tests/MomentoRedisClientTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+use Momento\Cache\Errors\InvalidArgumentError;
+use Momento\Cache\Errors\NotImplementedException;
+use Momento\Cache\MomentoCacheClient;
+use PHPUnit\Framework\TestCase;
+
+class MomentoRedisClientTest extends TestCase
+{
+    private static Redis $client;
+
+    /**
+     * Setup cache client before each class.
+     */
+    public static function setUpBeforeClass(): void
+    {
+        SetupIntegrationTest::setupIntegrationTest();
+        self::$client = SetupIntegrationTest::getClient();
+    }
+
+    /**
+     * @throws RedisException
+     */
+    public function testSetAndGetKeyValue(): void
+    {
+        $key = 'test_key';
+        $value = 'test_value';
```

Comment:
Throughout the tests, let's use randomly generated strings for keys to not collide. Can use a uuid.